### PR TITLE
fix/docs: Enum documentation fixes

### PR DESCRIPTION
**Pull request by bsian03**
# PULL REQUEST

## **Overview**

Fixes documentation related to accessing enum properties

## **Status**

- [X] Typings have been updated or don't need to be.
- [X] This PR have been tested and is ready to be merged.

## **Semantic versioning classification**

- [ ] This PR introduces *BREAKING* changes.
- [ ] This PR adds new features, improve the code and implies minimal changes.
- [X] This PR fixes a bug and references the relevant issue or documentation.
- [ ] This PR improve performance or code refactor without API changes.
- [X] This PR **only** includes non-code changes (documentation, CI, tools...).

### CODE REVIEW

File: src/Utility/Constants/DiscordEnums.js [hunk 1]
```diff
@@ -150,43 +152,43 @@ export const DISCORD_PERMISSIONS = [
...
+ * MANAGE_ROLES: 268435456,
+ * MANAGE_WEBHOOKS: 536870912,
+ * MANAGE_EMOJIS: 1073741824,
+ * ALL: 2147483647,
  * }}
  * @readonly
  * @enum {Number}
  */
-const PERMISSIONS_NUMBERS = {
+export const PERMISSIONS_NUMBERS = {
```
Comment (Khaazz) [inline src/Utility/Constants/DiscordEnums.js]: how will this behave with PERMISSION_NUMBERS.ALL being dynamically added after the export?

Comment (bsian03) [reply src/Utility/Constants/DiscordEnums.js]: Don't think it'll make a difference? It's referenced in the object export anyways

Comment (bsian03) [reply src/Utility/Constants/DiscordEnums.js]: JSDoc doesn't show without it there though

Comment (Khaazz) [reply src/Utility/Constants/DiscordEnums.js]: I don't think it makes a difference either but it would be worth trying.
I'll test it before merging.

Comment (Khaazz) [reply src/Utility/Constants/DiscordEnums.js]: Yes it indeed work as expected! No Problem here!

**Pull request merged**